dvc/repo/experiments/executor/local.py

```diff
@@ -213,3 +215,58 @@ def cleanup(self, infofile: str):
             checkpoint = self.scm.get_ref(EXEC_CHECKPOINT)
             if checkpoint and checkpoint != self._orig_checkpoint:
                 self.scm.set_ref(EXEC_APPLY, checkpoint)
+
+    @classmethod
+    def save(
+        cls,
+        info: "ExecutorInfo",
+        is_checkpoint: bool = False,
+        force: bool = False,
+    ) -> ExecutorResult:
+        from dvc.repo import Repo
+
+        exp_hash: Optional[str] = None
+        exp_ref: Optional[ExpRefInfo] = None
+
+        dvc = Repo(os.path.join(info.root_dir, info.dvc_dir))
```

Comment:
Why create a `Repo` instead of receiving as arg?

Reply:
Executors are set up to do it this way (without ever passing complex classes like `Repo`) so that they work easily in multiprocessing environments. We can't pass `Repo` because it's not pickle-able.